src/utils/collection.ts

```diff
@@ -122,6 +122,9 @@ function resolveSource(source: string | CollectionSource | CollectionSource[] |
     }
 
     if (source.repository) {
+      if (source.repository.includes('bitbucket')) {
```

Comment:
```suggestion
      if (source.repository.startsWith('https://bitbucket.org/')) {
```

It it better to check for full domain, to prevent matching urls like `https://github.com/foo/bitbucket-exporter`